packages/frontend/web-ui/src/app/store/features/authSlice.js

```diff
@@ -1,6 +1,35 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { createAuthByCredentials } from '../thunk/createAuthByCredentials';
 import { createAuthByToken } from '../thunk/createAuthByToken';
 
+function createAuthPendingReducer(state) {
+  state.status = 'checking';
+  state.token = null;
+  state.errorMessage = null;
+}
+
+function createAuthFulfilledReducer(state, action) {
+  state.status = 'not-authenticated';
+
+  if (action.payload.statusCode === 200) {
```

Comment:
Maybe a switch block would be more appropiate

Reply:
Done!